plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -99,7 +99,15 @@ const MemberComponent = ({ member }: { member: UserEntity }) => {
                 {displayName}
               </Link>
             </Typography>
-            <Typography variant="caption">{profile?.email}</Typography>
+            {profile?.email && (
+              <Link
+                href={`mailto:${profile.email}`}
+                target="_blank"
+                rel="noreferrer noopener"
```

Comment:
Could you try to change the import of `Link` in this file to instead use the one from `@backstage/core-components`? That one is _supposed_ to be used everywhere (but apparently wasn't in this file), and it is meant to set `target` and `rel` automatically and correctly so you could remove these here hopefully. The other benefit is that it'll be connected to analytics, styling etc etc in a cross-app way.

Reply:
replaced react-router link with Link component from `@backstage/core-components`